probers/pidfile/api.go

```diff
@@ -0,0 +1,30 @@
+package pidfile
+
+import (
+	"github.com/Symantec/tricorder/go/tricorder"
+	"os"
+)
+
+type pidconfig struct {
+	processname   string
+	pidfilepath   string
+	process       os.Process
```

Comment:
This does not seem to be used to preserve useful state between calls to Probe(), so it's simpler and cleaner to just create a local variable in the code that uses it.